content/doc/book/glossary/index.adoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: chapter
+---
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+:notitle:
+
+[glossary]
+= Glossary
+
+
+[glossary]
+
+Freestyle Job::
+    A Freestyle Job
```

Comment:
@rtyler If you're trying to define something and having trouble, try thinking about it as it appears in the wild--who is using it? What do they use it for? What does it do that is unique within the product or feature set? What's the business case for it, ie why do your users care about it?